help/styles/styles.css

```diff
@@ -513,6 +505,85 @@ main .section:empty {
   pointer-events: none;
 }
 
+main .section.rating {
+  display: block;
+}
+
+main .section.rating .default-content-wrapper {
+  margin: unset;
+  width: unset;
+}
+
+main .section.rating .default-content-wrapper h3 {
+  font-size: 1.25rem;
+  margin-bottom: 20px;
+  font-family: var(--button-font-family);
+}
+
+main .section.rating .default-content-wrapper ul {
+  list-style-type: none;
+  padding-left: 0;
+  font-size: 1rem;
+}
+
+main .section.rating .default-content-wrapper ul li {
+  margin-top: 10px;
+  margin-bottom: 10px;
```

Comment:
I see on the original that the `li` has higher spacing:
```
margin: 16px 0;
padding: 8px 0;
```

Reply:
I updated the margin to 16px, looks better visually now